pkg/client/client.go

```diff
@@ -1017,9 +1017,9 @@ func (c *ConmonClient) metadataBytes(ctx context.Context) ([]byte, error) {
 	return metadata, nil
 }
 
-// CreateaNamespacesConfig is the configuration for calling the
+// CreateNamespacesConfig is the configuration for calling the
```

Comment:
do we need type aliases to keep backward compatibility?

Reply:
though I don't know if we're using this anywhere so it may not actually matter..

Reply:
No right now it’s not being used.